programmers/난이도별/level02.주식가격/SSSOy.java

```diff
@@ -0,0 +1,17 @@
+class Solution {
+    public int[] solution(int[] prices) {
+        int[] answer = new int[prices.length];
+        
+        for(int i = 0; i < prices.length - 1; i++) {
+            for(int j = i + 1; j < prices.length; j++) {
```

Comment:
```suggestion
        int pricesLen = prices.length;
        int[] answer = new int[pricesLen];
        
        for(int i = 0; i < pricesLen - 1; i++) {
            for(int j = i + 1; j < pricesLen; j++) {
```
`prices.length`는 변하지 않는 값이니 첫 줄에 선언해서 **한 번만 계산**하도록 한 뒤 그 값을 가져와서 사용하는 게 더 깔끔할 수 있을 것 같아요. 아랫줄에 있는 `.length`도요! (요건 제가 JS를 하면서 익힌 버릇인데 별 상관없다면 참고만 해주세요...ㅎ..)